test/services/cache/ExportsCacheModelBuilder.scala

```diff
@@ -117,6 +117,20 @@ trait ExportsCacheModelBuilder {
         BorderTransport(borderModeOfTransportCode, meansOfTransportOnDepartureType, meansOfTransportOnDepartureIDNumber)
       )
     )
+  def withConsignmentReferences(consignmentReferences: Option[ConsignmentReferences]): CacheModifier =
+    _.copy(consignmentReferences = consignmentReferences)
+
+  def withLocations(locations: Locations): CacheModifier =
+    _.copy(locations = locations)
+
+  def withParties(parties: Parties): CacheModifier =
+    _.copy(parties = parties)
+
+  def withPreviousDocumentsData(previousDocumentsData: Option[PreviousDocumentsData]): CacheModifier =
```

Comment:
For Optional stuff, below I've done a `withX(...)` method and a `withoutX()` method

It means less visual clutter in unit tests (which at the best of times are fairly unreadable)

Reply:
Will bear in mind for future but am leaving this as is for now.